packages/convert-svg-to-png/bin/convert-svg-to-png

```diff
@@ -28,9 +28,13 @@ const { CLI } = require('convert-svg-core');
 
 const PNGProvider = require('../src/PNGProvider');
 
+
 (async() => {
   const cli = new CLI(new PNGProvider());
-
+  process.on('unhandledRejection', (e) => {
+    cli.error(`convert-svg-to-png failed: ${e.stack}`);
+    process.exit(1);
+  })
```

Comment:
I don't really like the idea of letting errors bubble up this high. If it's not catchable, I'd rather not through anything.